### ISSUE TITLE
"environmental package" and GOLD ecosystem terms are limited to soil for all 12 MIxS packages

### ISSUE THREAD

**Issue by pvangay** [microbiomedata/sheets_and_friends#123]
This happens regardless of what MIxS package I load. The other terms in the template seem appropriate for the selected environmental package -- but it seems to be that perhaps the allowable values here are just restricted incorrectly?

![image](https://user-images.githubusercontent.com/7374085/165582067-a8445524-b49e-403f-b7e9-0b93c0055bb6.png)

![image](https://user-images.githubusercontent.com/7374085/165582109-d3685937-53a1-4a2c-855e-945812639bb1.png)

**Comment by turbomam** [comment]
Yes, I'm aware of that. We discussed it a little in some meeting. I can't remember who was there or what decision we came to. I think I'm supposed to leave the GOLD path pulldowns empty for all package templates. If you have input I can still act on it. I'm more likely to have success with soemthing that's global, not tuned for individual templates.

@pkalita-lbl is experimenting with solutions for pulldowns that would be dependent on the values in other pulldowns.

Back to my meeting...

**Comment by pvangay** [comment]
Best to leave these values empty for all other packages except Soil. I think you did this for the EnvO terms right? users may think that they don't have the correct template loaded.

it's it's too hard to enable it for soil only, i would just remove it for all packages given the release being so soon.

**Comment by turbomam** [comment]
@pvangay I am going to start by changing these globally. The MIxS environmental triad columns and the GOLD path columns will all be blank and require strings that match their patterns.

I will try to add the previous support for pulldowns back into all of the soil related templates but don't know how that will goo.

**Comment by turbomam** [comment]
GOLD paths: edit [modifications_long tab on nmdc-dh-sheets-mam-non-anno](https://docs.google.com/spreadsheets/d/1cMlPKgjZh-v21aMYCm9x1TxzE5BwGQptBQcvuaYAtC8/edit#gid=1241218612)

specifically, the ranges for 

- ecosystem
- ecosystem_category
- ecosystem_subtype
- ecosystem_type
- specific_ecosystem

from enums like `ecosystem_enum` to `string`

**Comment by turbomam** [comment]
@pvangay 

Check this out at https://microbiomedata.github.io/sheets_and_friends/linkml.html

The previous GOLD pulldowns are now active for soil templates but disabled for all others. We can curate lists for each column per each other environment, but there's noting in place yet guaranteeing that the columns values go together in a reasonable way.

**Comment by turbomam** [comment]
The MIxS environmental triad columns are all free-form for all enviroment templates now. Tomorrow I'll let you know what else I can pull off by Friday.

**Comment by turbomam** [comment]
Oh shoot, the first part of this issue is about the _environment_ column, not the MIxS columns. There's another issue for that somewhere isn't there?

**Comment by pvangay** [comment]
GOLD terms look good. but "environment package" is still showing soil only.
![image](https://user-images.githubusercontent.com/7374085/165637522-0ae671d3-d278-461f-bb99-01c5d6051f23.png)

This is the other issue: https://github.com/microbiomedata/sheets_and_friends/issues/118 -- enable EnvO triad dropdowns for the soil templates. if you're able to also keep dropdowns for GOLD ecosystem path for soil templates too - that'd be great. but we agreed that for now, it's ok to remove dropdowns for all templates until you are able to find a way to make it work. thanks mark!
